cdap-kubernetes/pom.xml

```diff
@@ -76,6 +76,10 @@
       <artifactId>logback-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.twill</groupId>
+      <artifactId>twill-core</artifactId>
```

Comment:
twill-core has quite a lot of dependencies. If you are only using one class, better to exclude all the dependencies. The Resources class in Twill has no dependency on non JDK classes

Reply:
Done. Excluded all dependencies of twill-core